storage/reads/datatypes/storage_common.proto

```diff
@@ -252,5 +252,6 @@ message ReadWindowAggregateRequest {
   TimestampRange range = 2 [(gogoproto.nullable) = false];
   Predicate predicate = 3;
   int64 WindowEvery = 4;
+  int64 Offset = 6;
   repeated Aggregate aggregate = 5;
```

Comment:
is there a reason that these are out of order of their enum(?) values? Just curious.

Reply:
Not really. I was just going off the field ordering of the procedure spec.